src/NzbDrone.Core/Notifications/NotificationService.cs

```diff
@@ -72,6 +74,16 @@ private string GetMessage(Series series, List<Episode> episodes, QualityModel qu
                                     qualityString);
         }
 
+        private string GetMessage(Series series, bool deleteFiles)
+        {
+            string deleteMessage = deleteFiles ? 
+                "Removed from Sonarr and disk" : 
```

Comment:
We should update to match whatever we land on in `src/NzbDrone.Core/Notifications/Discord/Discord.cs`

Reply:
Better yet we should pass in the message from one place and re-use it.

Reply:
Resolved in upcoming commit - connected it to SeriesDeleteMessage